packages/dds/tree/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -12,7 +12,8 @@ import { deepFreeze } from "../../utils";
 import { cases, TestChangeset } from "./utils";
 
 const type: TreeSchemaIdentifier = brand("Node");
-const tomb = "Dummy Changeset Tag";
+const detachedBy = undefined;
+const detachIndex = 0;
```

Comment:
Can we rename these to something more meaningful? Alternatively, just inline these values.

Also, why do all the tests use `undefined` as the value for `deatchedBy` fields?

Reply:
They used undefined because I had not updated the test coverage to make sure `compose` does the right thing with that values. I have now down that and inlined the index part.